src/api/app/controllers/webui/package_controller.rb

```diff
@@ -4,22 +4,22 @@ class Webui::PackageController < Webui::WebuiController
   include Webui::ManageRelationships
   include BuildLogSupport
 
+  NON_BINARY_BUILD_RESTULS = ['rpmlint.log', '_buildenv'].freeze
```

Comment:
```suggestion
  NON_BINARY_BUILD_RESULTS = ['rpmlint.log', '_buildenv'].freeze
```

Reply:
This reminds me, I wanted to turn this around...

Reply:
done